### PR TITLE
Comment

### PR DESCRIPTION
**Pull request by niranjanneeru**

### CODE REVIEW

File: src/pages/taskDetails/api.ts [hunk 1]
```diff
@@ -6,11 +6,27 @@ const taskDetailApi = baseApi.injectEndpoints({
...
+      }),
+      providesTags: ['task.detail']
+    }),
+    addComments: builder.mutation({
+      query: ({ id, body }) => ({
+        url: `/tasks/${id}/comments`,
+        method: 'post',
+        body
+      }),
+      invalidatesTags: ['task.detail']
```
Comment (Vaishnav-NT) [inline src/pages/taskDetails/api.ts:18]: use constants

File: src/pages/taskDetails/api.ts [hunk 2]
```diff
@@ -6,11 +6,27 @@ const taskDetailApi = baseApi.injectEndpoints({
       query: (id) => ({
         url: `/tasks/${id}`,
         method: 'get'
+      }),
+      providesTags: ['task.detail']
```
Comment (Vaishnav-NT) [inline src/pages/taskDetails/api.ts:10]: use constants

File: src/services/index.ts [hunk 3]
```diff
@@ -14,7 +14,7 @@ const baseApi = createApi({
   refetchOnMountOrArgChange: true,
   refetchOnReconnect: true,
   endpoints: () => ({}),
-  tagTypes: ['employee.list', 'task.list']
+  tagTypes: ['employee.list', 'task.list', 'task.detail']
```
Comment (Vaishnav-NT) [inline src/services/index.ts:17]: constants

**Review by Vaishnav-NT — COMMENTED** [review]
Change tagtypes to constants

**Review by snehajoy05 — APPROVED** [review]
approved

**Pull request merged**